src/requirements/data/exams/ExamCredit.ts

```diff
@@ -1,16 +1,17 @@
-import { AppUser } from '../../../user-data';
+import { AppUser, FirestoreAPIBExam } from '../../../user-data';
 import { CourseTaken } from '../../types';
 
 export type ExamRequirements = {
   readonly name: string;
   readonly fulfillment: {
-    readonly courseEquivalents: Record<string, number>;
+    readonly courseEquivalents?: Record<string, number | number[]>;
```

Comment:
why this instead of just `number[]`

Reply:
A large majority of them have a single equivalent course and I didn't want to write the brackets for every exam. It adds some complexity in the function, but makes the data easier to read and modify imo. Do you think I should change it to just `number[]`?

Reply:
I think so, having a consistent data format is ideal, especially when we completely control the data and have the freedom to change them.